src/utils/tracer.ts

```diff
@@ -1,5 +1,7 @@
 import tracer from "dd-trace"
 
-tracer.init()
+tracer.init({
+  sampleRate: 1,
```

Comment:
sample rate = 1 means? every 1ms?

Reply:
from the docs 
> Controls the ingestion sample rate (between 0.0 and 1.0) between the Agent and the backend.

Reply:
they provide more extensive documentation [here](https://docs.datadoghq.com/tracing/trace_pipeline/ingestion_mechanisms/?tab=nodejs)